kevlar/tests/test_call.py

```diff
@@ -107,13 +107,13 @@ def test_variant_kmers():
 
 
 @pytest.mark.parametrize('part,coord,window', [
-    (12, 7027087, 'CAGGGAGAGGCAGCCTGCCCTCAACCTGGGAGAGCACTGTCTAATCAGCTCCCATCTCA'
+    (12, 7027071, 'CAGGGAGAGGCAGCCTGCCCTCAACCTGGGAGAGCACTGTCTAATCAGCTCCCATCTCA'
```

Comment:
This was changed in the last month or so, and I don't remember having a good reason for doing so. I must have introduced a bug with the handling of contig left overhangs. This change (and the one below) revert to the original correct values.